content/en/certificates.md

```diff
@@ -3,78 +3,102 @@ title: Chain of Trust
-    * [Let's Encrypt Authority X4 (Signed by ISRG Root X1)](/certs/letsencryptauthorityx4.pem.txt)
+  * Let's Encrypt Authority X4
+    * [signed by ISRG Root X1](/certs/letsencryptauthorityx3.pem.txt)
+    * [cross-signed by IdenTrust](/certs/lets-encrypt-x4-cross-signed.pem.txt)
+  * Let's Encrypt R4
+    * [signed by ISRG Root X1](/certs/lets-encrypt-r4.pem.txt)
+    * [cross-signed by IdenTrust](/certs/lets-encrypt-r4-cross-signed.pem.txt)
+  * Let's Encrypt E2
+    * [signed by ISRG Root X2](/certs/lets-encrypt-e2.pem.txt)
 * Retired
-  * [Let's Encrypt Authority X2 (IdenTrust cross-signed)](/certs/lets-encrypt-x2-cross-signed.pem.txt)
-    * [Let's Encrypt Authority X2 (Signed by ISRG Root X1)](/certs/letsencryptauthorityx2.pem.txt)
-  * [Let's Encrypt Authority X1 (IdenTrust cross-signed)](/certs/lets-encrypt-x1-cross-signed.pem.txt)
-    * [Let's Encrypt Authority X1 (Signed by ISRG Root X1)](/certs/letsencryptauthorityx1.pem.txt)
+  * Let's Encrypt Authority X1
+    * [signed by ISRG Root X1](/certs/letsencryptauthorityx1.pem.txt)
+    * [cross-signed by IdenTrust](/certs/lets-encrypt-x1-cross-signed.pem.txt)
+  * Let's Encrypt Authority X2
+    * [signed by ISRG Root X1](/certs/letsencryptauthorityx2.pem.txt)
+    * [cross-signed by IdenTrust](/certs/lets-encrypt-x2-cross-signed.pem.txt)
 
 # Cross Signing
 
-Our intermediate “Let’s Encrypt Authority X3” represents a single public/private
+Each of our intermediates represents a single public/private
 key pair. The private key of that pair generates the signature for all end-entity
 certificates (also known as leaf certificates), i.e. the certificates we issue
 for use on your server.
 
-Our intermediate is signed by ISRG Root X1. ISRG's root is widely trusted at this
-point, but our intermediate is still cross-signed by IdenTrust's "DST Root CA X3"
+Our RSA intermediates are signed by ISRG Root X1. ISRG's root is widely trusted at this
+point, but our intermediates are still cross-signed by IdenTrust's "DST Root CA X3"
 (now called "TrustID X3 Root") for additional client compatibility. The IdenTrust
 root has been around longer and thus has better compatibility with older devices
-and operating systems (e.g. Windows XP). [Download "TrustID X3 Root" on
-identrust.com](https://www.identrust.com/support/downloads) (or, alternatively,
-you can download a copy here: [.pem](/certs/trustid-x3-root.pem.txt),
-[.p7b](/certs/trustid-x3-root.p7b)).
+and operating systems (e.g. Windows XP, Android 7). You can [download "TrustID X3 Root" from
+IdenTrust](https://www.identrust.com/support/downloads) (or, alternatively,
+you can [download a copy from us](/certs/trustid-x3-root.pem.txt)).
 
 Having a cross-signature means there are two sets of intermediate certificates
-available, both of which represent our intermediate. One is signed by DST Root
+available, both of which represent each RSA intermediate. One is signed by DST Root
```

Comment:
"both of which represent each" is a little confusing. Let's say instead:

> Having cross-signatures means each of our RSA intermediates has two versions representing the same signing key. One is signed by DST Root....

Reply:
I like it, done.